lxd/container.go

```diff
@@ -329,6 +331,15 @@ func containerLXDCreateInternal(
 		"Container created in the DB",
 		log.Ctx{"container": name, "id": id})
 
+	myConfig := map[string]string{}
+	if err := shared.DeepCopy(&args.Config, &myConfig); err != nil {
+		return nil, err
+	}
+	myDevices := shared.Devices{}
+	if err := shared.DeepCopy(&args.Devices, &myDevices); err != nil {
+		return nil, err
+	}
```

Comment:
Can you split it in a separate commit? Should make it simpler to merge/rebase as @mikemccracken is working on fixes for the #996 fallout.